core/controllers/ingest.py

```diff
@@ -153,18 +153,18 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
     return build_success_response(programme_metadata=programme_metadata, do_load=do_load)
 
 
-def parse_body(body: dict) -> tuple[str, int, dict | None, bool]:
+def parse_body(body: dict) -> tuple[str, int, dict | None, bool, str | None, str | None]:
     """Parses the request body.
 
     :param body: request body
     :return: parsed values
     """
-    fund = body.get("fund_name")
-    reporting_round = body.get("reporting_round")
+    fund: str = str(body.get("fund_name"))
```

Comment:
This is a bit risky - if `fund_name` is not present, we'll end up with a string `"None"` rather than a real `None` value

Reply:
I think we just need to codify what's actually happening here in the return value rather than change the code logic?

ie `fund: str | None = body.get("fund_name")`

Reply:
Changed it as discussed.